support/python/mkdocstrings_handlers/cxx/__init__.py

```diff
@@ -50,63 +62,71 @@ def doxyxml2html(nodes: List[et.Element]):
     out += '<code class="language-cpp">' if tag == 'pre' else ''
     if n.text:
       out += escape_html(n.text)
-    out += doxyxml2html(n)
+    out += doxyxml2html([n])
```

Comment:
Looks like this introduces infinite recursion: https://github.com/fmtlib/fmt/actions/runs/10303709064/job/28524343469?pr=4110

Reply:
Should probably be `list(n)` instead of `[n]` because what we need is a list of chile elements.

Reply:
> Should probably be `list(n)` instead of `[n]` because what we need is a list of chile elements.

Oh my! I had introduced the `from __future__ import annotations` so that I could use the operator `|` (I tested this in a `python:3.8` docker), but then I also pushed this change (something I was trying on my machine). Sorry about that! I hope it works now.